hooked/views.py

```diff
@@ -72,9 +73,12 @@ def check_webhook_signature(self, request):
             return
 
         self.log_failure(
-            'Mismatch between computed and claimed signature of incoming '
-            'events. I computed {}, but the HTTP header said I should '
-            'expect to find {}'.format(computed_signature, claimed_signature))
+            message='Mismatch between computed and claimed signature of incoming events',
+            extra={
+                'expected_signature': computed_signature,
+                'actual_signature': claimed_signature
+            }
+        )
```

Comment:
This probably makes these variables visible in Sentry, but if you're just logging to console/disk I wonder if you really see the `expected_signature` and `actual_signature` in this case.

Really what Sentry wants, though, is `extra={'data': {... your data here... }}` but I don't know if that is too Sentry-specific when we're trying to make something broadly useful. (see [Sentry docs](https://docs.sentry.io/clients/python/integrations/logging/#usage))